### DISCUSSION TITLE
posts/pay-system/

### DISCUSSION THREAD

**Discussion by giscus** [team-json-delivery/team-json-delivery.github.io#16] [comment]
# posts/pay-system/

💸 결제 시스템은 정확히 무엇을 말하는걸까요?! 이번 블로그에서의 결제 시스템은 신용 카드, 은행 카드 등을 연동하여 결제 서비스를 제공하는 시스템을 말합니다. 요즘은 결제 시스템을 사용하시는 분들이 많기 때문에 ‘안정, 유연, 확장 가능’을 고려하여 시스템을 구축해야 합니다. 하지만 무엇보다도 ‘작은 실수로도 상당한 금액 손실이 발생’할 수 있다는 점을 간과해서는 안 됩니다. 💻 결제 시스템 흐름! 결제 시스템을 알기 위해서는 ‘대금 수신 흐름’을 알아야 합니다. 대금 수신 흐름 대금 수신 흐름은 다음과 같이 진행되게 됩니다.

https://team-json-delivery.github.io/posts/pay-system/

**Comment by zoster0828** [comment]
diagram들이 정말 명확하네요!
